code/modules/mob/living/simple_animal/bot/construction.dm

```diff
@@ -347,6 +347,44 @@
 					S.name = created_name
 					qdel(src)
 
+
+//Honkbot Assembly
+
+
+
+/obj/item/honkbot_assembly
+	name = "incomplete honkbot assembly"
+	desc = "The clown's up to no good once more"
+	icon = 'icons/mob/aibots.dmi'
+	icon_state = "honkbot_arm"
+	item_state = "honkbot_arm"
+	var/build_step = 0
+	var/created_name = "lil' Honkie"
+
+
+/obj/item/honkbot_assembly/attackby(obj/item/I, mob/user, params)
+
+	if(isprox(I) && (build_step == 0))
```

Comment:
make a define for the build steps(you can also add an #undef at the end of the file/after this attackby)